crypto/src/mls/conversation/conversation_guard.rs

```diff
@@ -0,0 +1,63 @@
+// this isssue is about creating this helper struct,
+// but we'll see the dead code warning until we actually use it in follow-up work
```

Comment:
Just reading this comment, it's not clear what `this issue` refers to (I'm guessing the related Jira issue).
I'd prefer something like `Temporarily silence the compiler, until we use ConversationGuard in follow-up work`.

Reply:
Yes, but given that the PR removing it is in progress and probably going to be ready for review today, could we proceed anyway?